src/app/auth/components/authorisation.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { FormGroup, FormBuilder, FormControl, Validators } from '@angular/forms';
+
+
+@Component({
+  selector:"authorisation",
+  templateUrl: "authorisation.component.html",
+  styleUrls: ["authorisation.component.scss"]
+})
+
+export class AuthorisationComponent {
+
+  public greenEye: boolean;
+  public loginForm: FormGroup;
+
+  @ViewChild('input') el: ElementRef;
+
+  constructor(private _builder: FormBuilder) {
+    this.greenEye = false;
+    this.loginForm = this._builder.group(
+      {
+        'name': [localStorage.getItem('name'), [
+          Validators.required,
+          Validators.minLength(5)]
+        ],
+        'password': [localStorage.getItem('password'), [
+          Validators.required,
+          Validators.minLength(8)]
+        ]
+      });
+  }
+
+  public login() {
+    localStorage.setItem('password', this.loginForm.controls['password'].value);
```

Comment:
Stop using access for controls like here. Please, have a look in NG Docs on ReactiveForms!